### PR TITLE
feat : input

### PR DESCRIPTION
**Pull request by sudipkundu999**
File structure:
components>input>input.css => contains CSS for input only
docs>component-presentation>input-presentation.css => contains CSS for presenting the input on the documentation site

P.S. some colors are used as "red", "green" intentionally and will be changed during refactor

### CODE REVIEW

File: Docs/component-presentation/input-presentation.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.input-display-container pre code::-webkit-scrollbar {
+    display: none;
+}
+
+
+@media screen and (min-width:1000px) {
+    .input-display{
+        max-width: 720px;
```
Comment (purohitdheeraj) [inline Docs/component-presentation/input-presentation.css]: Is it intentional that you have used px inside media queries or 
should be in rem unit?

**Review by purohitdheeraj — APPROVED** [review]
code Looks Great Sudip 👍
you can merge it

File: Docs/component-presentation/input-presentation.css (hunk 1 above)
Comment (sudipkundu999) [reply Docs/component-presentation/input-presentation.css]: This is by mistake. It should be in rems. Fixing it.

**Comment by itsmebipul00** [comment]
good to merge

**Review by developer-anand — APPROVED** [review]
Good Work Man, good to merge.

**Review by zoef-yt — APPROVED** [review]
Good to merge.  Nice work.

**Pull request merged**